### PR TITLE
Clarifications and additional links to CRD and cross-links to documentation

### PR DESCRIPTION
**Pull request by roberto-mello**
Clarifications to docs about restoring individual databases, plus additional links to CRD and cross-linking to improve readbility

**Checklist:**

 
 - [x] Have you added an explanation of what your changes do and why you'd like them to be included?
 - [x] Have you updated or added documentation for the change, as applicable?
 - [x] Have you tested your changes on all related environments with successful results, as applicable?
   - [ ] Have you added automated tests?

**Type of Changes:**

 
 - [ ] New feature
 - [ ] Bug fix
 - [x] Documentation
 - [ ] Testing enhancement
 - [ ] Other

**What is the current behavior (link to any open issues here)?**

Documentation can use improvements.

**What is the new behavior (if this is a feature change)?**
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Other Information**:

### CODE REVIEW

File: docs/content/tutorial/disaster-recovery.md [hunk 1]
```diff
@@ -231,7 +231,19 @@ you can start accessing your database from that specific point in time!
...
+or to move selected databases to a machine that does not have enough space to restore the
+entire cluster backup.
+
+pgBackRest supports this case, but it is important to make sure this is what you want.
+Restoring in this manner will restore the requested database from backup and making it
+accessible, but all of the other databases in the backup will **not** be accessible after restore.
+
+For example, if your backup includes databases `test1` and `test2`, and you request that `test2`
+be restored, the `test1` database will not be accessible. Please review the pgBackRest documentation
+on the [limitations on restoring individual databases](https://pgbackrest.org/user-guide.html#restore/option-db-include).
```
[the remark this answers is not in the document]
Comment (roberto-mello) [reply docs/content/tutorial/disaster-recovery.md]: It does make sense to add it to a Warning block, now that you mention it. I'll update the PR with the change. Thanks for looking into it

File: docs/content/tutorial/disaster-recovery.md [hunk 2]
```diff
@@ -231,7 +231,22 @@ you can start accessing your database from that specific point in time!
...
+You might need to restore specific databases from a cluster backup, for performance reasons
+or to move selected databases to a machine that does not have enough space to restore the
+entire cluster backup.
+
+{{% notice warning %}}
+pgBackRest supports this case, but it is important to make sure this is what you want.
+Restoring in this manner will restore the requested database from backup and make it
+accessible, but all of the other databases in the backup will NOT be accessible after restore.
+
+For example, if your backup includes databases `test1`, `test2` and `test3`, and you request that
```
Comment (benjaminjb) [inline docs/content/tutorial/disaster-recovery.md:243]: ⛏️ (nitpick, not essential). I'm not sure what the rest of our docs look like, but I slightly prefer
```suggestion
For example, if your backup includes databases `test1`, `test2`, and `test3`, and you request that
```

**Review by benjaminjb — APPROVED** [review]
LGTM, I like the links from the docs to other parts of the docs/crd

File: docs/content/tutorial/disaster-recovery.md (hunk 2 above)
Comment (roberto-mello) [reply docs/content/tutorial/disaster-recovery.md]: Added this change

**Pull request merged**